types/backbone-relational/index.d.ts

```diff
@@ -122,7 +122,19 @@ declare module 'backbone' {
 
     }
 
-    export class Store extends Events {
+    export class Store implements Events {
+        on(eventName: string, callback?: Function, context?: any): any;
+        on(eventMap: EventsHash): any;
+        on(eventName: any, callback?: any, context?: any): any;
+        off(eventName?: string, callback?: Function, context?: any): any;
+        trigger(eventName: string, ...args: any[]): any;
+        bind(eventName: string, callback: Function, context?: any): any;
+        unbind(eventName?: string, callback?: Function, context?: any): any;
+        once(events: string, callback: Function, context?: any): any;
+        listenTo(object: any, events: string, callback: Function): any;
```

Comment:
As discussed [here](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes), please don't use the type `Function`.

Reply:
Fixed.  Thank you.  These had been propagated from the original implementation in Backbone, so I fixed it there and all the other locations as well.